apps/nextjs-website/src/components/molecules/RequiredTextField/RequiredTextField.tsx

```diff
@@ -60,7 +63,7 @@ const RequiredTextField: FC<RequiredTextFieldProps> = ({
       onChange={onChange}
       onBlur={() => setIsDirty(true)}
       sx={{
-        backgroundColor: 'white',
+        backgroundColor: palette.common.white,
```

Comment:
```suggestion
        backgroundColor: palette.background.paper,
```
What are the drawbacks of this? (And, take this suggestion valid for all the other `backgroundColor`)